Header header
float32 angleRad
float32 distNormalized
float32 speedXKmh

